packages/cypress/src/executors/cypress/schema.json

```
@@ -178,6 +178,17 @@
     "runnerUi": {
       "type": "boolean",
       "description": "Displays the Cypress Runner UI. Useful for when Test Replay is enabled and you would still like the Cypress Runner UI to be displayed for screenshots and video."
+    },
+    "autoCancelAfterFailures": {
+      "oneOf": [
+        {
+          "type": "number"
+        },
+        {
+          "type": "boolean"
+        }
+      ],
+      "description": "Specify the number of failures to cancel a run being recorded to the Cloud or `false` to disable auto-cancellation."
```

Let's be more explicit in the description about which "Cloud" to avoid potential confusion with Nx Cloud. Also, the boolean value can only be `false`, so let's allow only that value.
```suggestion
        {
          "type": "boolean",
          "const": false
        }
      ],
      "description": "Specify the number of failures to cancel a run being recorded to the Cypress Cloud or `false` to disable auto-cancellation."
```